tests/test_tulok_boed.py

```
@@ -0,0 +1,88 @@
+from datetime import datetime
+from os.path import dirname, join
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.tulok_boed import TulokBoedSpider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "tulok_boed.json"),
+    url="https://tulsaschools.diligent.community/Services/MeetingsService.svc/meetings?from=2024-12-01&to=9999-12-31",  # noqa
```

Ruff (0.14.8)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)